deps/chakrashim/include/v8-debug.h

```diff
@@ -91,7 +91,7 @@ class V8_EXPORT Debug {
   static Local<Context> GetDebugContext(Isolate* isolate);
 
   static void EnableDebug();
-  static void EnableInspector();
+  static void EnableInspector(bool enableReplayDebug = false);
```

Comment:
Seems to remember that it was present when I implemented debugger support. If not I am fine with adding new function.